exercises/two-fer/src/test/java/TwoferTest.java

```diff
@@ -38,4 +38,16 @@ public void anotherNameGiven() {
 
         assertEquals(expected, twofer.twofer(input));
     }
+
+    /* Track specific */
```

Comment:
I don't know if there is a convention for denoting track-specific tests, but this seems fine.

#733 didn't have it, though.

Reply:
I like being explicit about it so that it does not get removed without (at least) looking at the context of why it was added.